docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -0,0 +1,503 @@
+
+To prepare an example supporting factory data, add a partition called ``factory_data`` in a pm_static file. The partition size should be a multiple of one flash page (for nRF52 and nRF53 SoCs, a single page size equals 4 kB).
+
+See the following code snippet for an example of creating a factory data partition in the ``pm_static.yml`` file.
+The snippet is based on the ``pm_static.yml`` file from the [Lock application example](../../examples/lock-app/nrfconnect/configuration/nrf52840dk_nrf52840/pm_static_dfu.yml) and uses the nRF52840 DK:
+
+
+```
+...
+mcuboot_primary_app:
+    orig_span: &id002
+        - app
+    span: *id002
+    address: 0x7200
+    size: 0xf3e00
+
+factory_data:
+    address: 0xfb00
+    size: 0x1000
+    region: flash_primary
+
+settings_storage:
+    address: 0xfc000
+    size: 0x4000
+    region: flash_primary
+...
+```
+
+In this example, a ``factory_data`` partition has been placed between the **application partition** (mcuboot_primary_app) and the **settings storage**. Its size has been set to one Flash page (4 kB). 
+
+Use Partition Manager's report tool to ensure you created a factory data partition correctly. To do that, navigate to the example directory and run the following command
+
+```
+$ west build -t partition_manager_report
+```
+
+The output will look similar to the following one:
+
+```
+
+  external_flash (0x800000 - 8192kB):
++---------------------------------------------+
+| 0x0: mcuboot_secondary (0xf4000 - 976kB)    |
+| 0xf4000: external_flash (0x70c000 - 7216kB) |
++---------------------------------------------+
+
+  flash_primary (0x100000 - 1024kB):
++-------------------------------------------------+
+| 0x0: mcuboot (0x7000 - 28kB)                    |
++---0x7000: mcuboot_primary (0xf4000 - 976kB)-----+
+| 0x7000: mcuboot_pad (0x200 - 512B)              |
++---0x7200: mcuboot_primary_app (0xf3e00 - 975kB)-+
+| 0x7200: app (0xf3e00 - 975kB)                   |
++-------------------------------------------------+
+| 0xfb000: factory_data (0x1000 - 4kB)            |
+| 0xfc000: settings_storage (0x4000 - 16kB)       |
++-------------------------------------------------+
+
+  sram_primary (0x40000 - 256kB):
++--------------------------------------------+
+| 0x20000000: sram_primary (0x40000 - 256kB) |
++--------------------------------------------+
+
+```
+
+### Generating factory data partition
+
+To store the factory data set in the device's persistent storage, convert the data from the JSON file to its binary representation in the CBOR format.
+To do this, use the [nRF Connect script](../../scripts/tools/nrfconnect/nrfconnect_generate_partition.py) to generate factory data partition.
```

Comment:
```suggestion
To do this, use [nrfconnect_generate_partition.py](../../scripts/tools/nrfconnect/nrfconnect_generate_partition.py) to generate factory data partition.
```
I think using specific script names rather than "nRF Connect script" would be more readable.